src/modules/universal-discovery/components/bookmarks/bookmarks.component.js

```diff
@@ -23,6 +23,32 @@ export default class BookmarksComponent extends Component {
         this.bookmarksContent();
     }
 
+    /**
+     * Called when bookmark was removed
+     *
+     * @method onBookmarkRemoved
+     * @param {Object} itemLocation
+     * @memberof BookmarksComponent
+     */
+    onBookmarkRemoved(itemLocation) {
+        this.removeItem(itemLocation);
+    }
+
+    /**
+     * Removes item from list
+     *
+     * @method removeItem
+     * @param {Object} itemToRemoveLocation
+     * @memberof BookmarksComponent
+     */
+    removeItem(itemToRemoveLocation) {
+        const areTheSameItems = (fstItemLocation, sndItemLocation) => fstItemLocation.id === sndItemLocation.id;
```

Comment:
`areSameItems`, then `(a, b) => a.id === b.id;`

Reply:
I think that `fstItemLocation` is more precise in this case; you know you should pass `item`'s `Location`, not the `item`.